cfme_perf.sh

```diff
@@ -7,12 +7,37 @@ else
   YUM=yum
 fi
 
-sudo $YUM install -y python-virtualenv gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel libcurl-devel redhat-rpm-config  gcc-c++  freetype-devel libffi-dev
+sudo $YUM install -y numpy \
+                     python-virtualenv \
+                     gcc \
+                     postgresql-devel \
+                     libxml2-devel \
+                     libxslt-devel \
+                     libcurl-devel \
+                     redhat-rpm-config \
+                     gcc-c++ \
+                     python-virtualenv \
+                     libffi-devel \
+                     libpng \
+                     libpng-devel \
+                     freetype-devel \
+                     freetype-devel \
+                     openssl-devel
+
+echo "### Creating Virtual Environment: .cfme_perf"
 virtualenv .cfme_perf
+
+echo "### Activating .cfme_per"
 echo "export PYTHONPATH='`pwd`'" | tee -a ./.cfme_perf/bin/activate
 echo "export PYTHONDONTWRITEBYTECODE=yes" | tee -a ./.cfme_perf/bin/activate
-
 . ./.cfme_perf/bin/activate
+
+echo "### Upgrading pip"
+pip install --upgrade pip
```

Comment:
For rhel we should have easy_install pip before this right?  (I assume centos would need that as well?)